mne_bids_pipeline/steps/freesurfer/_01_recon_all.py

```diff
@@ -77,21 +87,34 @@ def main(*, config) -> None:
 
     You must have freesurfer available on your system.
 
-    Run via the MNE BIDS Pipeline's `run.py`:
+    Run via the MNE BIDS Pipeline's CLI:
 
-    python run.py --steps=freesurfer --config=your_pipeline_config.py
+    mne_bids_pipeline --steps=freesurfer --config=your_pipeline_config.py
 
     """  # noqa
     subjects = get_subjects(config)
+    sessions = get_sessions(config)
     root_dir = config.bids_root
     subjects_dir = Path(get_fs_subjects_dir(config))
     subjects_dir.mkdir(parents=True, exist_ok=True)
 
     with get_parallel_backend(config.exec_params):
         parallel, run_func = parallel_func(run_recon, exec_params=config.exec_params)
         parallel(
-            run_func(root_dir, subject, fs_bids_app, subjects_dir)
+            # TODO How can we infer if inner `for session in sessions` loop is needed?
+            #      Should we specify in `config`? As written, it will *only* work for
+            #      cases where subjects_dir has this layout:
```

Comment:
I think it's okay to use some filesystem inspection using whatever conditionals you need to construct the subj-session pairs, and then call the `parallel`. So something like
```
subj_session = list()
for subject in subjects:
    if mri_is_root_level(subject):  # TODO: Need to write this helper to do the filesystem inspection part!
        subj_session.append((subject, None))
    else:
        subj_session.extend((subject, session) for session in sessions)
parallel(run_func(...) for subject, session in subj_session)
```
where `session=None` would mean "root-level not session specific". Does that make sense? If not maybe I misunderstand the problem.